pkgs/by-name/an/ankama-launcher/package.nix

```
@@ -0,0 +1,44 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+let
+  pname = "ankama-launcher";
+  version = "3.12.17";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://launcher.cdn.ankama.com/installers/production/Ankama%20Launcher-Setup-x86_64.AppImage";
+    hash = "sha256-aD5YYcye/WPpb0Kir55qCL+RGbK8ZSdmi4NKmqKkW/Y=";
+  };
+
+  appimageContents = appimageTools.extract { inherit name src; };
+in
+
+appimageTools.wrapType2 rec {
```

You don't need `rec`
```suggestion
appimageTools.wrapType2 {
```